scripts/check-events

```diff
@@ -6,12 +6,32 @@ require 'find'
 # These members/tags are common to multiple events
 BYTE_SIZE_COUNT = ['byte_size', 'count']
 
+# SUFFIX => [MESSAGE, COUNTERS, ADDITIONAL_TAGS]
+EVENT_CLASSES = {
+  'BytesReceived' => [
+    'Bytes received.', ['received_bytes'], ['byte_size'], ['protocol']
+  ],
+  'EventsReceived' => [
+    'Events received.', ['received_events', 'received_event_bytes'], ['count', 'byte_size']
+  ],
+  'EventsSent' => [
+    'Events sent.', ['sent_events', 'sent_event_bytes'], ['count', 'byte_size']
+  ],
+  'BytesSent' => [
+    'Bytes sent.', ['sent_bytes'], ['byte_size'], ['protocol']
+  ],
```

Comment:
I'm surprised this works. These items have different numbers of elements in the value list.

Reply:
ruby is... surprising...

Reply:
This will need to be corrected still.

Reply:
done!

Reply:
my bad